docs/whirlpool/docusaurus.config.js

```diff
@@ -38,6 +41,16 @@ export default {
     ],
   ],
 
+  stylesheets: [
+    {
+      href: 'https://cdn.jsdelivr.net/npm/katex@0.13.24/dist/katex.min.css',
```

Comment:
Do we want to have a cdn link? Is it also in the `rehype-katex` package? If so we can reference that in `node_modules`. I see it has a checksum which would prevent any remote modification I guess so not strictly necessary

Reply:
I can't seem to find it. I could copy the css code in `docs/whirlpool/static/katex.min.css. What do you think?